React/Base/RCTJavaScriptLoader.mm

```diff
@@ -274,9 +274,11 @@ static void attemptAsynchronousLoadOfBundleAtURL(NSURL *scriptURL, RCTSourceLoad
 
     // Validate that the packager actually returned javascript.
     NSString *contentType = headers[@"Content-Type"];
-    if (![contentType isEqualToString:@"application/javascript"] &&
-        ![contentType isEqualToString:@"text/javascript"]) {
-      NSString *description = [NSString stringWithFormat:@"Expected Content-Type to be 'application/javascript' or 'text/javascript', but got '%@'.", contentType];
+    NSArray<NSString *> *contentTypeArr = [contentType componentsSeparatedByString:@";"];
```

Comment:
Please don't use abbreviations.
`NSString *mimeType = [[contentType componentSeparatedByString:@";"] firstObject]` should be enough, right?